Index: neophyte/Views/Registration/RegisterPage.xaml.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>using System;\r\nusing System.Collections.Generic;\r\nusing System.Linq;\r\nusing System.Text;\r\nusing System.Threading.Tasks;\r\n\r\nusing Xamarin.Forms;\r\nusing Xamarin.Forms.Xaml;\r\n\r\nnamespace neophyte.Views.Registration\r\n{\r\n    [XamlCompilation(XamlCompilationOptions.Compile)]\r\n    public partial class RegisterPage : ContentPage\r\n    {\r\n        public RegisterPage()\r\n        {\r\n            InitializeComponent();\r\n        }\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- neophyte/Views/Registration/RegisterPage.xaml.cs	(revision b7535566feb63c51824535796687098395292fad)
+++ neophyte/Views/Registration/RegisterPage.xaml.cs	(date 1599044438504)
@@ -1,9 +1,3 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
 using Xamarin.Forms;
 using Xamarin.Forms.Xaml;
 
